201-dsc-linux-azure-storage-on-ubuntu/install-resource.parameters.json

```diff
@@ -1,38 +1,39 @@
 {
-  "username": {
-    "value": "yourusername"
-  },
-  "password": {
-    "value": "yourpassword"
-  },
-  "location": {
-    "value": "West US"
-  },
-  "newStorageAccountName": {
-    "value": "uniquestorageaccount"
-  },
-  "vmName": {
-    "value": "yourvmname"
-  },
-  "vmSize": {
-    "value": "Standard_D1"
-  },
-  "ubuntuOSVersion": {
-    "value": "14.04.3-LTS"
-  },
-  "mode": {
-    "value": "Install"
-  },
-  "storageAccountName": {
-    "value": "yourstorageaccountname"
-  },
-  "storageAccountKey": {
-    "value": "yourstorageaccountkey"
-  },
-  "containerName": {
-    "value": "yourcontainername"
-  },
-  "resourceZipFileName": {
-    "value": "yourmodule_version.zip"
-  }
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
```

Comment:
DSCForLinux extension has multiple configuration settings for different functionality, so I used mulitple parameter files to explain the different usage. 
Just found that with the refactor in 2.0 version, some parameter files can be removed, I will remove the duplicated parameter files.